public/stylesheets/local.css

```diff
@@ -186,10 +186,10 @@ table td > .badge {
 }
 
 /* Used to give table rows the same padding as Bootstrap card headers */
-.card table tr td:first-child, .card table tr th:first-child {
+.card table:not(.table-no-row-padding) tr td:first-child, .card table:not(.table-no-row-padding) tr th:first-child {
```

Comment:
An alternative fix might be to change to `.card > table` to restrict to tables that are immediate descendants of cards? I think that's the case that @astrosticks was worried about in #2290?

Reply:
That might be cleaner, let me try that.

Reply:
Changed, I also added `.card .table-responsive > table` since sometimes there's an extra div in the chain.